tox.ini

```diff
@@ -14,6 +14,11 @@ minversion = 4.22.0
 labels =
     freezedeps = freezedeps-print,freezedeps-py{3.13,3.12,3.11,3.10,3.9,3.8}
 
+[testenv:install-venv]
+envdir = {toxinidir}/.venv
+allowlist_externals = echo
+commands = echo "Virtualenv installed (via tox) at {envdir}"
```

Comment:
Nice!

I _think_ this is inheriting something from the bare `testenv` environment that I want to double-check if you haven't already: I think it's installing a packaged version of the SDK, rather than an editable version.

If it's editable, you should see a `.pth` file in `.venv/lib/python3.13/site-packages/` that contains a path. If it isn't, you'll just see `globus_sdk/` in that directory, and that will lead to unexpected results.